test/index.js

```diff
@@ -55,3 +57,23 @@ test('fetchConfigFromReq - strip non-standard keys', (t) => {
   t.end()
 
 })
+
+test('createInfuraMiddleware - runs through all the retry attempts due to empty response', (t) => {
```

Comment:
We need a test for when `null` is returned by Infura, b/c that's what's breaking this.